pkgs/by-name/sc/scons/package.nix

```
@@ -14,6 +14,7 @@ python3Packages.buildPythonApplication rec {
 
   patches = [
     ./env.patch
+    ./etc_paths.d.patch
```

It is recommended to prefer `fetchpatch` over adding files to nixpkgs:

```suggestion
    # https://github.com/SCons/scons/pull/4603
    (fetchpatch {
      name = "etc_paths.d.patch";
      url = "https://github.com/SCons/scons/commit/2d5e3a40a613225b329776ab9dbd9abcd2d24222.patch";
      hash = "sha256-N1xQOvsPTi7a2maEZJQVu6vJ9AoWMqDOsScXHp9KuXI="; # Please compute the hash.
    })
```

---

Good point!